### DISCUSSION TITLE
An error has appeared on the babel homepage

### DISCUSSION THREAD

**Discussion by tt-a1i** [babel/babel#16985] [General]
![image](https://github.com/user-attachments/assets/754d468c-05b7-4f95-936e-9e5945611080)
https://babeljs.io/

**Comment by JLHwung** [comment]
`$` is defined by the jQuery library, which is loaded from https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js. Please check if you can access this URL.
